File: video-editor/models/messages/media_processing_completed.py
from typing import Optional
from models.broker_model import BrokerModel


class MediaProcessingCompleted(BrokerModel):
    media_id: str
    success: bool
    thumb_tiny_key: Optional[str] = None
    thumb_medium_key: Optional[str] = None
    error: Optional[str] = None

File: media-ingest/src/Contracts/MediaProcessingCompleted.cs
using System.Text.Json.Serialization;

namespace Contracts;

public class MediaProcessingCompleted
{
    [JsonPropertyName("mediaId")]
    public Guid MediaId { get; set; }

    [JsonPropertyName("success")]
    public bool Success { get; set; }

    [JsonPropertyName("error")]
    public string? Error { get; set; }
}
